test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -349,19 +352,17 @@ static Set<URL> parseClassPathWithSymlinks() throws Exception {
     public static void ensureInitialized() {}
 
     /**
-     * Temporarily dsiables security manager for a test.
-     *
-     * <p> This method is only callable by {@link org.elasticsearch.test.ESTestCase}.
+     * Temporarily disables security manager for a test.
      *
      * @return A closeable object which restores the test security manager
      */
     @SuppressWarnings("removal")
     public static Closeable disableTestSecurityManager() {
```

Comment:
We could change this to return `Releaseable` so we don't need to catch `IOException` in the methods below?